css2/tables.html

```diff
@@ -996,7 +996,7 @@ <h4>17.5.2.2 <a name="auto-table-layout">Automatic table layout</a></h4>
   formatted content may span any number of lines but may not overflow
   the cell box. If the specified <a href="visudet.html#propdef-width" class="noxref"><span
   class="propinst-width">'width'</span></a> (W) of the cell is greater
-  than MCW, W is the minimum cell width. A value of 'auto' means that
+  than MCW, the minimum cell widthis set to W. A value of 'auto' means that
```

Comment:
"widthis" typo

Reply:
gah, used the github editor, and it javascripted on me whilst typing. Will fix.